include/nuttx/spinlock.h

```diff
@@ -51,6 +51,11 @@ typedef uint8_t spinlock_t;
 
 #include <arch/spinlock.h>
 
+#if defined(CONFIG_RW_SPINLOCK)
```

Comment:
where you define CONFIG_RW_SPINLOCK?

Reply:
It should be added to Kconfig but I didn't do that.

Reply:
Done.

Reply:
change to:
```
#ifdef CONFIG_RW_SPINLOCK
```